lazybytez/jojo-discord-bot
# Update ping module to be less disturbing

## Description
Update ping module to be less disturbing

## Related issue
Resolves #117

## How can this be tested?
 - Ensure that `ping` and `pong` chat messages no longer trigger a response from the bot
 - Ensure that the `/ping` and `/pong` command now return ephemeral responses
